invenio_github/tasks.py

```diff
@@ -71,37 +70,40 @@ def release_default_exception_handler(release, ex):
 ]
 
 
-@shared_task(max_retries=6, default_retry_delay=10 * 60, rate_limit='100/m')
+@shared_task(max_retries=6, default_retry_delay=10 * 60, rate_limit="100/m")
 def disconnect_github(access_token, repo_hooks):
```

Comment:
I am not sure what to do with this task, it should probably be a post processor on the disconnect signal emitted by invenio-oauthclient